microsoft/testsuites/core/lsvmbus.py

```diff
@@ -104,3 +107,149 @@ def lsvmbus_count_devices_channels(self, node: Node) -> None:
+            "out_write_index",
+            "state",
+            "uevent",
+            "vendor",
+        ]
+        vmbus_channel_files = [
+            "cpu",
+            "events",
+            "in_mask",
+            "interrupts",
+            "out_mask",
+            "read_avail",
+            "write_avail",
+        ]
+        cmd_result = node.execute(
+            f"ls -lt {device_path}/driver_override",
+            sudo=True,
+            shell=True,
+        )
+        # driver_override file not existing - contain recent commits
+        if 0 != cmd_result.exit_code:
+            vmbus_driver_files.append(
+                [
+                    "monitor_id",
+                    "server_monitor_conn_id",
+                    "server_monitor_latency",
+                    "server_monitor_pending",
+                    "client_monitor_conn_id",
+                    "client_monitor_latency",
+                    "client_monitor_pending",
+                ]
+            )
+            vmbus_channel_files.append(["latency", "pending"])
+        for file in vmbus_driver_files:
+            node.execute(
+                f"ls -lt {device_path}/{file}",
+                sudo=True,
+                shell=True,
+                expected_exit_code=0,
+                expected_exit_code_failure_message=(
+                    f"{device_path}/{file} doesn't exist"
+                ),
+            )
+        cmd_result = node.execute(
+            f"ls -d {device_path}/channels/*", shell=True, sudo=True
+        )
+        channel = cmd_result.stdout
+        for file in vmbus_channel_files:
+            node.execute(
+                f"ls -lt {channel}/{file}",
+                sudo=True,
+                shell=True,
+                expected_exit_code=0,
+                expected_exit_code_failure_message=(f"{channel}/{file} doesn't exist"),
+            )
+        in_out_files = [
+            "intr_mask",
+            "read_bytes_avail",
+            "write_bytes_avail",
+        ]
+        for in_out_file in in_out_files:
+            in_file_value = node.tools[Cat].read(
+                f"{device_path}/in_{in_out_file}", force_run=True, sudo=True
+            )
+            out_file_value = node.tools[Cat].read(
+                f"{device_path}/out_{in_out_file}", force_run=True, sudo=True
+            )
+            assert_that(in_file_value).described_as(
+                f"value of {device_path}/in_{in_out_file} is not equal to"
+                f" {device_path}/out_{in_out_file}"
+            ).is_equal_to(out_file_value)
+
+        origin_interrupts = node.tools[Cat].read(
+            f"{channel}/interrupts", force_run=True, sudo=True
+        )
+        origin_events = node.tools[Cat].read(
+            f"{channel}/events", force_run=True, sudo=True
+        )
+        timeout = 60
+        timer = create_timer()
+        while timeout > timer.elapsed(False):
+            current_interrupts = node.tools[Cat].read(
```

Comment:
Is it possible some interrupts are not increased?

Reply:
tested many times, I didn't see it failed in this step.